src/python_testing/TC_EWATERHTRM_1_2.py

```diff
@@ -0,0 +1,109 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs: run1
```

Comment:
This needs to be added to `.github/workflows/tests.yaml` to get executed

Reply:
Should I do this change under https://github.com/project-chip/connectedhomeip/pull/34351 and assuming this PR(34347) has been merged prior to that as the test needs support from the all clusters app?

Reply:
Added now.

Reply:
Marking as resolved.